.npmrc

```diff
@@ -1,3 +1,3 @@
 registry=https://nexus.ci.diabeloop.eu/repository/npm-diabeloop/
 email=jenkins@diabeloop.fr
-_auth=${NEXUS_TOKEN}
+//nexus.ci.diabeloop.eu/repository/npm-diabeloop/:_auth=${NEXUS_TOKEN}
```

Comment:
why do we need to specify the registry here ?
is it a new requirement ?

Reply:
https://github.com/npm/cli/issues/3130
Fixed, but it should apparently be done that way